lib/licenses.nix

```diff
@@ -1164,6 +1164,13 @@ lib.mapAttrs mkLicense (
       fullName = "Sendmail License";
     };
 
+    sel = {
+      fullName = "Stalwart Enterprise License 1.0 (SELv1) Agreement";
+      url = "https://github.com/stalwartlabs/mail-server/blob/main/LICENSES/LicenseRef-SEL.txt";
+      free = false;
+      redistributable = false;
+    };
```

Comment:
Given that this is a bespoke license for a single package, would it make more sense to just inline it? Otherwise these are going to end up being most of licenses.nix.

Reply:
Done